admin/c2cgeoportal_admin/views/layers_wms.py

```diff
@@ -50,19 +50,25 @@ def index(self):
     def grid(self):
         return super().grid()
 
-    @view_config(route_name='c2cgeoform_action',
+    @view_config(route_name='c2cgeoform_item',
                  request_method='GET',
                  renderer='../templates/edit.jinja2')
     def view(self):
         return super().edit()
 
-    @view_config(route_name='c2cgeoform_action',
+    @view_config(route_name='c2cgeoform_item',
                  request_method='POST',
                  renderer='../templates/edit.jinja2')
     def save(self):
         return super().save()
 
-    @view_config(route_name='c2cgeoform_action',
+    @view_config(route_name='c2cgeoform_item',
                  request_method='DELETE')
     def delete(self):
         return super().delete()
+
+    @view_config(route_name='c2cgeoform_item_action',
+                 request_method='GET',
+                 renderer='../templates/edit.jinja2')
```

Comment:
There is no filter on duplicate action, maybe we should add a match_param filter here ?

Reply:
True, it works because at this time there is one and only one action